server/controllers/adminController.js

```diff
@@ -37,5 +37,55 @@ export default class AdminController {
         res.status(500).send(err.message);
       });
   }
+  /**
+   * @description -  admin approve requests
+   * @static
+   *
+   * @param {object} req - HTTP Request
```

Comment:
It's better to use `request` and `response` instead of `req` and `res`

Reply:
Alright, I will make appropriate changes, thank you